.flake8

```diff
@@ -20,6 +20,28 @@ per-file-ignores =
     # F841 local variable assigned to but never used
     # E741 ambiguous variable name 'l'
     __init__.py: F401, F403
+    # HTML
+    man/build_html.py: E501
+    # wxgui
+    doc/python/m.distance.py: E501
+    gui/scripts/d.wms.py: E501
+    gui/wxpython/image2target/g.gui.image2target.py: E501
+    gui/wxpython/photo2image/g.gui.photo2image.py: E501
+    gui/wxpython/animation/g.gui.animation.py: E501
+    gui/wxpython/tplot/g.gui.tplot.py: E501
+    gui/wxpython/iclass/g.gui.iclass.py: E501
+    gui/wxpython/mapdisp/test_mapdisp.py: E501
+    gui/wxpython/mapswipe/g.gui.mapswipe.py: E501
+    gui/wxpython/timeline/g.gui.timeline.py: E501
+    scripts/r.semantic.label/r.semantic.label.py: E501
+    scripts/g.extension/g.extension.py: E501
+    scripts/r.unpack/r.unpack.py: E501
+    scripts/v.unpack/v.unpack.py: E501
+    scripts/v.import/v.import.py: E501
```

Comment:
These are already down below around line 230.

Reply:
The other line has E501 already, so there is no reason to add that.

Reply:
Now I understand what you mean. File names in the flake8 configuration file must be unique. The duplicate file names re-state the issues and I think that causes the CI to fail.

I tried to keep the fixes consistent with those in E501. You can find the notes in the top thread. I am still learning about Git and collaboration practices. Your patience is greatly appreciated.

Thank you so much for fixing the title.